src/components/common/SectionTitle/SectionTitle.tsx

```diff
@@ -0,0 +1,21 @@
+import { SerializedStyles } from '@emotion/react';
+import { FC } from 'react';
+import { fontSizes } from 'src/styles/Tokens';
+
+export type SectionTitleProps = {
+  children: string;
```

Comment:
[imo]
`children` だとHTML要素を期待しているのかな？と呼び出し側で思われそうなので、`title` とか `text` とかが良いかなと思いました！

```suggestion
  title: string;
```

Reply:
children の型は入力しないと教えてくれないので気持ちはわかりますが、 children である意味は使う際に

```tsx
<SectionTitle>
XXXXX
</SectionTitle>
```

で書けるようにするためなので children は個人的には OK です！

Reply:
ありがとうございます！
たしかにchildrenで受け取る理由はあまりなく、分かりやすさ重視でtitleにしようと思います！

Reply:
これ全然イメージしてなかった！
この使い方を想定しているなら、childrenでもいいかもですね！

```
<SectionTitle>
XXXXX
</SectionTitle>
```

Reply:
```
<SectionTitle>
XXXXX
</SectionTitle>
```

こちらを想定していました！